kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/ext/TypeExtensions.kt

```diff
@@ -25,7 +25,6 @@ fun Type?.toKtVariantType(): ClassName = when {
         "godot.core.VariantType",
         fqName.substringAfterLast(".").camelToSnakeCase().uppercase(Locale.getDefault())
     )
-    fqName == "kotlin.Unit" -> ClassName("godot.core.VariantType", "NIL")
```

Comment:
Why do we remove this ?

Reply:
because we already have that on line 10: https://github.com/utopia-rise/godot-kotlin-jvm/blob/34fa5fd647a49c96c7f6d6b047f45728c7e8b4c8/kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/ext/TypeExtensions.kt#L10